application/app/views/projects/_project_header.html.erb

```diff
@@ -0,0 +1,80 @@
+<style>
+    [data-project-name-target="form"] {
+        min-height: 2.80rem;
+    }
+
+    [data-project-name-target="form"] input {
+        min-width: 400px;
+    }
+</style>
```

Comment:
I would prefer having this defined as a CSS class in a SCSS file

Reply:
yes - forgot to move it after completing debugging.

Reply:
fixed